### PR TITLE
Core and ext 1 done

### PR DESCRIPTION
**Pull request by ben-corbin**

### CODE REVIEW

File: src/Filters.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+function Filters(props) {
+  // A function that takes the argument e which is the event object from the listener, then checks if the value
```
Comment (Willcowen) [inline src/Filters.js:2]: I really like your use of comments, is this to help remind yourself and cement the learning on certain lines of code?

File: src/App.js [hunk 2]
```diff
@@ -2,44 +2,44 @@ import './styles/reset.css'
...
-  id: '001-beetroot',
-  name: 'beetroot',
-  price: 0.35
-}
-
-What should a cart item look like? 🤔
-*/
-
-console.log(initialStoreItems)
+import { useEffect, useState } from 'react'
```
Comment (Willcowen) [inline src/App.js:5]: Nice use of component extraction.

File: src/App.js [hunk 3]
```diff
@@ -2,44 +2,44 @@ import './styles/reset.css'
...
         <ul className="item-list store--item-list">
-          {/* Wrtite some code here... */}
+          <Store items={items} cart={cart} setCart={setCart} />
         </ul>
       </header>
       <main id="cart">
         <h2>Your Cart</h2>
         <div className="cart--item-list-container">
           <ul className="item-list cart--item-list">
-            {/* Wrtite some code here... */}
```
Comment (Willcowen) [inline src/App.js:34]: Perhaps consider removing these lines of code.

File: src/App.js [hunk 4]
```diff
@@ -2,44 +2,44 @@ import './styles/reset.css'
...
     <>
       <header id="store">
         <h1>Greengrocers</h1>
+        <Filters
+          setItems={setItems}
+          initialStoreItems={initialStoreItems}
+          items={items}
+        />
         <ul className="item-list store--item-list">
-          {/* Wrtite some code here... */}
```
Comment (Willcowen) [inline src/App.js:27]: Perhaps try to add some conditional rendering here, if you get a chance to try extension two!

**Comment by 7LGnGZkY** [comment]
Clear and concise

File: src/Filters.js (hunk 1 above)
Comment (ben-corbin) [reply src/Filters.js:2]: Yeah exactly, it helps with remembering my decisions and also semi-rubberducking myself with the longer winded logic.

**Pull request closed without merging**